src/figuresClasses.ts

```diff
@@ -1,19 +1,76 @@
 export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
 
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: 'triangle' = 'triangle';
 
-}
+  constructor(
+    public color: 'red' | 'green' | 'blue',
```

Comment:
Why don't you employ something like this?
![image](https://user-images.githubusercontent.com/33294729/181259508-8a2bcc6f-48a1-4bea-9811-5510991cdfd4.png)